apps/archive/autocomplete.py

```diff
@@ -0,0 +1,53 @@
+
+import locale
+import superdesk
+
+from flask import current_app as app, request
+from datetime import timedelta
+from superdesk.utils import ListCursor
+from superdesk.utc import utcnow
+from superdesk.errors import SuperdeskApiError
+
+
+class AutocompleteResource(superdesk.Resource):
+    item_methods = []
+    resource_methods = ['GET']
+
+
+class AutocompleteService(superdesk.Service):
+
+    allowed_fields = {
+        'slugline',
+    }
+
+    def get(self, req, lookup):
+        field = request.args.get('field', 'slugline')
+        language = request.args.get('language', app.config.get('DEFAULT_LANGUAGE', 'en'))
+
+        if not app.config.get('ARCHIVE_AUTOCOMPLETE'):
```

Comment:
It might be good to also check `ARCHIVE_AUTOCOMPLETE_DAYS == 0 and ARCHIVE_AUTOCOMPLETE_HOURS == 0`, as those conditions equates to this endpoint returning empty values all the time.

Also text translation for API errors are missing, i.e. `from flask_babel import _`